routes/routeStack.js

```diff
@@ -1,32 +1,77 @@
-import { createStackNavigator } from "react-navigation-stack";
+import { createStackNavigator, TransitionPresets } from "react-navigation-stack";
```

Comment:
Good job! One of you will have to explain me how it works inside the actual function please hehe

Reply:
🥇

Reply:
🙏